config.json

```diff
@@ -1,7 +1,7 @@
 {
   "language": "C#",
   "active": true,
-  "blurb": "C# is a modern, object-oriented language with lots of great features, such as type-inference and async/await. The official documentation is very extensive and well-written.",
+  "blurb": "C# is a modern, object-oriented language with lots of great features, such as type-inference and async/await. The tooling is excellent and the documentation is extensive and well-written.",
```

Comment:
I would suggest putting a comma after "excellent". It separates what we're trying to convey and keeps the extensive/well-written part together. Or potentially ".. and provides extensive, well-written documentation".

Definitely don't want to get into a grammar war as so much of it can be opinionated, but the double and is throwing me for a loop a little. :)